src/main/java/com/ebay/sonar/test/github/TestGithub.java

```diff
@@ -4,6 +4,29 @@
 
 	public void testJava(){
 		System.out.println("test");
+		 if ( 3 != 3 ) { // always false
+  	System.out.println("test");
+}
+if ( 3 == 3 && 3 == 3 ) { // if the first one is true, the second one is too
+ 	System.out.println("test");
+}
+if ( 3== 3 || 3 == 3) { // if the first one is true, the second one is too
+  	System.out.println("test");
+}
+
+if ( 5 == 5 && 5 == 5 ) { // if the first one is true, the second one is too
+ 	System.out.println("test");
+}
+if ( 5== 5 || 5 == 5) { // if the first one is true, the second one is too
+  	System.out.println("test");
+}
+if ( 5 == 5 && 5 == 5 ) { // if the first one is true, the second one is too
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Identical sub-expressions on both sides of operator "==" [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.vip.qa.ebay.com/coding_rules#rule_key=squid%3AS1764)
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Identical sub-expressions on both sides of operator "&&" [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.vip.qa.ebay.com/coding_rules#rule_key=squid%3AS1764)
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Identical sub-expressions on both sides of operator "==" [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.vip.qa.ebay.com/coding_rules#rule_key=squid%3AS1764)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Assign this magic number 5 to a well-named constant, and use the constant instead. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.vip.qa.ebay.com/coding_rules#rule_key=squid%3AS109)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Assign this magic number 5 to a well-named constant, and use the constant instead. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.vip.qa.ebay.com/coding_rules#rule_key=squid%3AS109)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) '(' is followed by whitespace. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.vip.qa.ebay.com/coding_rules#rule_key=checkstyle%3Acom.puppycrawl.tools.checkstyle.checks.whitespace.ParenPadCheck)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) ')' is preceded with whitespace. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.vip.qa.ebay.com/coding_rules#rule_key=checkstyle%3Acom.puppycrawl.tools.checkstyle.checks.whitespace.ParenPadCheck)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Assign this magic number 5 to a well-named constant, and use the constant instead. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.vip.qa.ebay.com/coding_rules#rule_key=squid%3AS109)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Assign this magic number 5 to a well-named constant, and use the constant instead. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.vip.qa.ebay.com/coding_rules#rule_key=squid%3AS109)
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.vip.qa.ebay.com/coding_rules#rule_key=squid%3ATrailingCommentCheck)